src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -218,6 +220,50 @@ public UnfilteredRowIterator executePartitionReadCommand(SinglePartitionReadComm
         return command.queryMemtableAndDisk(cfs, executionController);
     }
 
+    public RangeIterator buildIterator()
```

Comment:
I agree that it makes more sense for this code to live here than in Operation

Reply:
Oh, thanks for confirming. I was slightly on the fence with that. I moved it mostly to get access to some private QueryController stuff like QueryContext.